src/Tizen.NUI.Components/Theme/DefaultThemeCommon.cs

```diff
@@ -378,7 +378,7 @@ public Theme Create()
                 },
             });
 
-            theme.AddStyleWithoutClone("Tizen.NUI.Components.Page", new ViewStyle()
+            theme.AddStyleWithoutClone("Tizen.NUI.Components.ContentPage", new AppBarStyle()
```

Comment:
Please use ViewStyle instead of AppbarStyle.

Reply:
Thank you for your comment.
The code has been fixed.